jetty-websocket/javax-websocket-server-impl/src/test/java/org/eclipse/jetty/websocket/jsr356/server/BinaryStreamTest.java

```diff
@@ -63,9 +62,18 @@ public void prepare() throws Exception
         server.addConnector(connector);
 
         ServletContextHandler context = new ServletContextHandler(server, "/", true, false);
-        ServerContainer container = WebSocketServerContainerInitializer.configureContext(context);
-        ServerEndpointConfig config = ServerEndpointConfig.Builder.create(ServerBinaryStreamer.class, PATH).build();
-        container.addEndpoint(config);
+        WebSocketServerContainerInitializer.configure(context, (servletContext, container) ->
+        {
+            try
+            {
+                ServerEndpointConfig config = ServerEndpointConfig.Builder.create(ServerBinaryStreamer.class, PATH).build();
+                container.addEndpoint(config);
+            }
+            catch (DeploymentException e)
```

Comment:
I think it would be better to define our own interface rather than use BiConsumer, so that we can let it throw DeploymentException and avoid the boilerplate handling everywhere

Reply:
Done.